package.json

```diff
@@ -3,18 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ngrx/core": "^1.2.0",
+    "@ngrx/store": "^6.1.0",
```

Comment:
@bellangowi  Please remove `@ngrx/core` and `@ngrx/store` i do not think we are using these.

Reply:
Noted